tests/Makefile.am

```diff
@@ -42,6 +47,7 @@ MAINTAINERCLEANFILES = \
 
 edit_cmd = $(SED) \
     -e 's|@softokenpath[@]|$(softokenpath)|g' \
+    -e 's|@p11kitclientpath[@]|$(p11kitclientpath)|g' \
```

Comment:
if we use the same .in file we just need a "tokenpath" variable in the .in file and the edit script will replace it wth the "right" token in each oif the config files.